src/config/index.ts

```diff
@@ -76,7 +66,7 @@ function getEnv(): Env {
     return Env.PRODUCTION
   }
 
-  return Env.LOCAL
+  return Env.PRODUCTION
```

Comment:
I'm not sure we want production as default if there's anything misconfigured. I'd rather use DEV/STAGING config so as to make sure we only hit prod endpoints intentionally.